fixtures/_fixtures/warnings.py

```diff
@@ -0,0 +1,47 @@
+#  fixtures: Fixtures with cleanups for testing and convenience.
+#
+# Licensed under either the Apache License, Version 2.0 or the BSD 3-clause
+# license at the users choice. A copy of both licenses are available in the
+# project source as Apache-2.0 and BSD. You may not use this file except in
+# compliance with one of these two licences.
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under these licenses is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# license you chose for the specific language governing permissions and
+# limitations under that license.
+
+from __future__ import absolute_import
+
+__all__ = [
+    'WarningsCapture',
+    ]
+
+import warnings
+
+import fixtures
+
+
+class WarningsCapture(fixtures.Fixture):
+    """A fixture context manager that copies and restores the warnings filter
+       upon exiting the fixture.
```

Comment:
Could you follow PEP 287 here? Something like:

"""Capture warnings.

While WarningCapture is active, warnings will be captured by the fixture.
:attribute captures: A list of warning capture WarningMessage objects.
"""